File: wignusi.BE/wignusi.Domain/ReadModels/OrderRm.cs
namespace wignusi.Domain.ReadModels
{
    public record BookQuantity(
        string Title,
        int Quantity
        );

    public record OrderRm(
        int OrderId,
        string Name,
        string Details,
        string Status,
        string Email,
        string Phone,
        decimal Total,
        BookQuantity[] Books
        );
}

File: wignusi.UI/src/app/api/models/order-rm.ts
/* tslint:disable */
/* eslint-disable */
import { BookQuantity } from '../models/book-quantity';
export interface OrderRm {
  books?: null | Array<BookQuantity>;
  details?: null | string;
  email?: null | string;
  name?: null | string;
  orderId?: number;
  phone?: null | string;
  status?: null | string;
  total?: number;
}
